nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pprint
+import six
+import time
+
+import nixnet
+from nixnet import constants
+from nixnet import types
+from nixnet.system import system
+
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+def main():
+    with system.System() as my_system:
+        print(list(my_system.databases))
```

Comment:
Hmm, I'm torn.

Technically, when you iterate on a mapping, you get back the keys rather than the values.

DAQmx implemented a [sequence](https://github.com/ni/nidaqmx-python/blob/master/nidaqmx/_task_modules/channel_collection.py) that also accepts strings.  Not sure how well that fits here.

I'd say carry on but this is something we need to eventually resolve.